modules/aws/ssm_test.go

```diff
@@ -0,0 +1,26 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/aws"
```

Comment:
This line is the issue. Since we are already in package `aws`, there is no need to import this. I am not sure why you don't get the import error though. Perhaps you have a different version of `go`?

Reply:
@yorinasub17 Good catch. I guess `go1.12.4` was hiding my error. Easy fix though. Thanks.

Reply:
Thanks for the fix! I just kicked off the build. Let's see if it passes this time!